source/_components/withings.markdown

```diff
@@ -0,0 +1,203 @@
+---
+layout: page
+title: "Withings"
+description: "Instructions on how to integrate Withings health products within Home Assistant."
+date: 2019-03-02 21:20
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: withings.png
+ha_category: 
+  - Health
+  - Sensor
+ha_release: 0.93.0
+ha_iot_class: "Cloud Polling"
+---
+
+The `withings` sensor platform consumes data from various health products produced by [Withings](https://www.withings.com).
+
+## {% linkable_title Setup %}
+
+### {% linkable_title Perquisites %}
+These perquisites are necessary only when creating your Withings developer account. It is important your service meets these requirements as the Withings site will check when you attempt to create your account.
+- Home assistant is accessible through a publicly resolvable domain name. An ip address or localhost will not work.
+- Service must be running on port 80 or 443 with SSL certs.
+
+Side Note: Upon account creation, Withings checks your server's settings by sending an HTTP HEAD request to the callback uri. While not tested, it may be possible to setup a webserver that responds to that path to avoid having your server online.
```

Comment:
```suggestion
Side Note: Upon account creation, Withings checks your server's settings by sending an HTTP HEAD request to the callback url. While not tested, it may be possible to setup a webserver that responds to that path to avoid having your server online.
```